template/src/ontology/Makefile.jinja2

```diff
@@ -290,7 +288,8 @@ all_mappings: $(MAPPING_FILES)
 # ----------------------------------------
 
 OBO_REPORT = {% for x in project.robot_report.report_on|default(["edit"]) %} {% if x=="edit" %}$(SRC){% else %}{{ x }}{% endif %}-obo-report{% endfor %}
-REPORTS = $(OBO_REPORT)
+ALIGNMENT_REPORT = {% for x in project.robot_report.report_on|default(["edit"]) %} {% if x =="edit" %}$(SRC){% else %}{{ x }}{% endif %}-align-report{% endfor %}
```

Comment:
😱

Reply:
?

Reply:
Could you elaborate your concerns here?

Reply:
No concerns at all, just amaaaaazement!! That was a _fast turnaround_ even for you!